frontend/src/pages/EventPage/CreatePhysicalEventPage.js

```diff
@@ -0,0 +1,188 @@
+import React, { useReducer } from "react";
+import { useAuth } from "../../auth/useAuth";
+
+import {TextField, Typography, Stack} from "@mui/material";
+import GenericCardLayout from "../../layouts/GenericCardLayout";
+import ImageUploader from '../../components/ImageUploader';
+import LoadingButton from '../../components/LoadingButton';
+import MapSelectComponent from "../../components/MapSelectComponent"
+
+function CreatePhysicalEventForm() {
+
+  const [formInput, setFormInput] = useReducer(
+    (state, newState) => ({ ...state, ...newState }),
+    {
+      title: "",
+      description: "",
+      category: "",
+      eventPrice: "",
+      labels: "",
+      rules: "",
+      address: ""
+    }
+  );
+  const [selectedImage, setSelectedImage] = React.useState(null);
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [position, setPosition] = React.useState({lat: 41, lng: 29});
+
+
+  const formatFormInput = (posterId) => {
+    return {
+      eventInfo: {
+        title: formInput.title,
+        description: formInput.description,
+        category: formInput.category,
+        eventPrice: formInput.eventPrice,
+        labels: formInput.labels,
+        posterId: posterId
+      },
+      location: {
+        latitude: position.lat,
+        longitude: position.lng,
+        address: formInput.address
+      },
+      rules: formInput.rules
+    }
+  }
+
+  const { postRequestWithImage } = useAuth()
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    setIsLoading(true)
+    
+    postRequestWithImage(
+      "/api/event/physical",
+      selectedImage,
+      formatFormInput,
+      (data) => "/event/"+data.id
+    )
+  };
+  
+  const handleInput = event => {
+    const name = event.target.name;
+    const newValue = event.target.value;
+    setFormInput({ [name]: newValue });
+  };
+
+  return (
+    <>
+      <Typography variant="h5" component="h3">
+        New Physical Event
+      </Typography>
+
+      <form onSubmit={handleSubmit}>
+        <Stack>
+          <TextField
+            required
+            id="outlined-required"
+            label="Event Title"
+            name="title"
+            defaultValue={formInput.title}
+            onChange={handleInput}
+            sx = {{marginY: 1}}
+          />
+          <TextField
+            required
+            id="outlined-required"
```

Comment:
`multiline` prop should be added, and we can set max height limit with `maxRows` or min limit with `minRows` props